python-iml-manager.spec.in

```diff
@@ -356,6 +358,7 @@ fi
 %dir %attr(0755,nginx,nginx)%{manager_root}
 %dir %attr(0755,nginx,nginx)/var/log/chroma
 %ghost /etc/nginx/conf.d/chroma-manager.conf
+/etc/nginx/default.d/chroma-manager-redirect.conf
```

Comment:
Does this need attr?

Reply:
Nope, it will be root.root per global attr above